docs/getting-started/server/guide.md

```diff
@@ -326,10 +326,6 @@ You are now ready to build and run your development server.
 
 7.  Test that the Api service is alive by navigating to
     [http://localhost:4000/alive](http://localhost:4000/alive)
-8.  Connect a client to your local server by configuring the client’s Api and Identity endpoints.
-    Refer to
-    [https://bitwarden.com/help/article/change-client-environment/](https://bitwarden.com/help/article/change-client-environment/)
-    and the instructions for each client in the Contributing Documentation.
 
 :::info
 
```

Comment:
I think this `info` (and the following `note`) admonitions need to be moved down into the `Client connection` section.